.vsts-ci.yml

```diff
@@ -420,7 +420,7 @@ jobs:
           R_LIB_PATH=~/Rlib
           export R_LIBS=${R_LIB_PATH}
           mkdir -p ${R_LIB_PATH}
-          RDscript -e "install.packages(c('R6', 'data.table', 'jsonlite', 'knitr', 'markdown', 'Matrix', 'RhpcBLASctl'),  lib = '${R_LIB_PATH}', dependencies = c('Depends', 'Imports', 'LinkingTo'), repos = 'https://cran.rstudio.com', Ncpus = parallel::detectCores())" || exit 1
+          RDscript .ci/install-r-deps.R --build --test || exit 1
```

Comment:
Please remove
https://github.com/microsoft/LightGBM/blob/c9de57b09cf9293213fa41b423e2409ebd9ae5c1/.vsts-ci.yml#L417
https://github.com/microsoft/LightGBM/blob/c9de57b09cf9293213fa41b423e2409ebd9ae5c1/.vsts-ci.yml#L426

Reply:
done in https://github.com/microsoft/LightGBM/pull/6808/commits/dae117f3d94d6c49873e6759b4da1b0a8bd7ea0e#diff-3e528906c029b6309d1c7491be6e3dbeaca567e4dde8a6257e3efa1356024827